src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions.mdx

```diff
@@ -18,7 +18,7 @@ redirects:
 
 import queriesPolicyNewAlertCondition from 'images/queries-nrql_screenshot-crop_create-alert-condition.webp'
 
-import queriesMetricExplorer from 'images/queries-nrql_screenshot-crop_metrics-create-alert-condition.webp'
+import queriesMetricExplorerAnimation from 'images/queries-nrql_screenshot-crop_metrics-create-alert-condition.gif'
```

Comment:
I'm seeing that this is an animated gif. Just to make sure, have you asked in #doc_eng_help if we can use gif files? I understood that we have to use webp from now on, but maybe there's an exception with animated gifs.

Reply:
I just confirmed with Zach that .gifs are ok. Thank you!